.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code, and run tests
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Automated testing
+
+on:
+  push:
+    branches: [nj-stable-dev, nj-stable]
+  pull_request:
+    branches: [nj-stable-dev, nj-stable]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: frontend
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
```

Comment:
[sand|possible boulder] is it possible to upgrade this to node 22? Looks like 18 is only going to be maintained until this summer and 22 is the active version

https://nodejs.org/en/about/previous-releases